packages/persist-state/src/lib/exclude-keys.ts

```diff
@@ -0,0 +1,19 @@
+import { Store, StoreValue } from '@ngneat/elf';
+import { map } from 'rxjs/operators';
+
+export function excludeKeys<S extends Store, State extends StoreValue<S>>(
```

Comment:
Sorry for the hassle but now that I see it I think it's better to make it an operator:

```ts
store.pipe(omitKeys(['foo', 'bar']))
```